packages/acceptance-tests/pkg-tests-specs/sources/commands/config/get.test.js

```diff
@@ -3,9 +3,8 @@ describe(`Commands`, () => {
     test(
       `it should print the requested configuration value for the current directory`,
       makeTemporaryEnv({}, async ({path, run, source}) => {
-        await expect(run(`config`, `get`, `pnpShebang`)).resolves.toMatchObject({
-          stdout: `#!/usr/bin/env node\n`,
-        });
+        const {stdout} = await run(`config`, `get`, `pnpShebang`);
+        expect(stdout).toContain(`'#!/usr/bin/env node'`);
```

Comment:
We can't change this output when the value is a string - some programs expect `yarn config get <something>` to return the raw value, it would break compatibility with those tools. Ex: the [Vue CLI](https://github.com/vuejs/vue-cli/blob/1a0b59142aa8797810ca90705290d960a4ee6d1e/packages/%40vue/cli/lib/util/shouldUseTaobao.js#L51).